lib/ramble/ramble/workspace/workspace.py

```diff
@@ -1059,18 +1059,46 @@ def append_result(self, result):
 
         self.results.update(result)
 
+    def simlink_result(self, filename_base, latest_base, file_extension, overwrite=True):
+        """
+        Create simlink of result file so that results.latest.txt always points
+        to the most recent analysis. This clobbers the existing link
+        """
+        out_file = os.path.join(self.root, filename_base + file_extension)
+        latest_file = os.path.join(self.root, latest_base + file_extension)
+
+        if overwrite and os.path.exists(latest_file):
```

Comment:
It might be good to check `os.is_symlink`, and provide a CLI argument to control the overwrite behavior.

It would be sad to clobber some file the user named made `results.latest.txt` (within the workspace). Admittedly though, this is pretty unlikely.

Reply:
I initially thought we might want to expose the overwrite, but on reflection I don't think it's worth it (especially because you might want a 3-way semantic around clobber-none, clobber-symlinks, and clobber-all...)

Another thing I considered is only clobbering simlinks (instead of all files), I was 50/50 on which is better but eventually decided that given our original semantic was clobbering `results.txt` then clobbering files is fine too (and have no idea how simlinks on windows like platforms work..) 

It sounds like you prefer only clobbering if simlink?

Reply:
Yeah, I think I'd prefer it only clobber if it is a symlink. I think it's really unlikely that this will actually impact users, but I think I'd rather not accidentally clobber some file named `results.latest.txt` that the user copied in for some reason.

Reply:
Pushed an update to address this